File: src/main/java/com/lord/small_box/dtos/RequestComparationNoteDto.java
package com.lord.small_box.dtos;

import java.util.Calendar;
import java.util.List;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class RequestComparationNoteDto {

	private Calendar requestDate;
	
	private String fromOrganization;
	
	private String mainOrganization;
	
	private String requestCode;
	
	private String depositName;
	
	private List<ControlRequestComparationNoteDto> items;
}

File: front/small-box/src/app/models/requestComparationNoteDto.ts
import { NumberSymbol } from "@angular/common";
import { ControlRequestComparationNoteDto } from "./controlRequestComparationNoteDto";

export class RequestComparationNoteDto{
    receiverId!:number;
    requestDate!:Date;
    fromOrganization!:string;
    mainOrganization!:string;
    requestCode!:string;
    depositName!:string;
    items!:ControlRequestComparationNoteDto[];
}